packages/top-app-bar/src/mwc-top-app-bar-base.ts

```diff
@@ -36,8 +36,12 @@ export class TopAppBarBase extends BaseElement {
   @query('.mdc-top-app-bar')
   protected mdcRoot!: HTMLElement;
 
+  // _actionItemsSlot should have type HTMLSlotElement, but when TypeScript's
+  // emitDecoratorMetadata is enabled, the HTMLSlotElement constructor will
+  // be emitted into the runtime, which will cause an "HTMLSlotElement is
+  // undefined" error in browsers that don't define it (e.g. Edge and IE11).
```

Comment:
Good catch.

I wonder how hard it would be to disable `emitDecoratorMetadata` in google3. Potentially would have decent size savings.

Reply:
> I wonder how hard it would be to disable `emitDecoratorMetadata` in google3. Potentially would have decent size savings.

Yep!